app/controllers/v1/notice_of_disagreements_controller.rb

```diff
@@ -39,5 +37,16 @@ def create
     def error_class(method:, exception_class:)
       "#{self.class.name}##{method} exception #{exception_class} (NOD_V1)"
     end
+
+    def clear_in_progress_form(submitted_appeal_uuid)
+      ActiveRecord::Base.transaction do
+        AppealSubmission.create!(user_uuid: @current_user.uuid,
```

Comment:
Ah yes, this is how it is done in the other decision reviews I believe. If this was not being done here that is a great catch!